docs/examples/201/web-app-sql-database/main.bicep

```diff
@@ -19,13 +19,11 @@ param skuCapacity int {
   minValue: 1
   default: 1
 }
-param sqlAdministratorLogin string {}
+param sqlAdministratorLogin string 
```

Comment:
I think the test is failing because of whitespace at the end of this line

Reply:
How is it possible a single whitespace be the cause of the build failing?

Reply:
The tests are pretty strict on formatting. It runs the bicep formatter on all bicep files and expects the submitted code to look the same. This ensures that all examples are formatted the same way.